packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -0,0 +1,344 @@
+import { ISliderStyleProps, ISliderStyles, getFocusStyle, AnimationVariables, getGlobalClassNames } from '../..';
+import { HighContrastSelector } from '@uifabric/styling';
+
+const GlobalClassNames = {
+  root: 'ms-Slider',
+  container: 'ms-Slider-container',
+  showTransitions: 'ms-Slider-showTransitions',
+  slideBox: 'ms-Slider-slideBox',
+  line: 'ms-Slider-line',
+  thumb: 'ms-Slider-thumb',
+  active: 'ms-Slider-active',
+  inactive: 'ms-Slider-inactive',
+  value: 'ms-Slider-value'
+};
+
+export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
+
+  const { className, theme, vertical, buttonClassName, showValue, disabled } = props;
+  const { palette } = theme;
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  // @TODO(keco): Do we need to apply .rootIsDisabled / .rootIsEnabled classes for backwards-compat?
+  return ({
+    root: [
+      classNames.root,
+      {
+        userSelect: 'none',
+        selectors: {
+          '&.titleLabel': {
+            padding: 0
+          }
+        }
+      },
+      vertical && [
+        {
+          marginRight: 8
+        }
+      ],
+      disabled && [
+        {
+          selectors: {
+            '.thumb': {
+              borderColor: palette.neutralTertiaryAlt,
+              selectors: {
+                [HighContrastSelector]: {
+                  borderColor: 'GrayText'
+                }
+              }
+            },
+            // @TODO(keco): Is this OK or should we still target .activeSection
+            '.ms-Slider-active': {
```

Comment:
In the `.scss` we use a mixture of the pre-fixed classes and others like `.activeSection` to apply styles. Do we need to continue to use them with this refactor to keep backwards-compatibility or should we migrate all to the `ms-` prefix? cc: @dzearing

Reply:
I've left a few in-line comments because I'm unsure of what route to take w.r.t the above.

Reply:
As per Teams convo, need to refactor to leverage CSS modules as .scss file did.